guides/spin/cli/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: single
+title:  "Install and Configure Spin CLI"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Install `spin`
+
+One way to manage applications and pipelines as code is through [spin](http://github.com/spinnaker/spin).
```

Comment:
"...through [spin CLI](https://github.com/spinnaker/spin)."

Cause that's what we're calling it elsewhere.